README.md

```diff
@@ -13,42 +13,57 @@ you to [configure variables](https://docs.ansible.com/ansible/latest/user_guide/
 
 First, install ansible (>2.0)
 
-OSX:
-
-  brew install ansible
-
-Linux:
-
-  apt-get install ansible
-
+- OSX: `brew install ansible`
+- Linux: `apt-get install ansible`
 
 Add this line to your application's Gemfile:
 
 ```ruby
 gem 'subspace'
 ```
 
-Or install it yourself as:
+Or install it yourself from the command line: `$ gem install subspace`
+
+### Other Requirements
+
+- [JQ](https://formulae.brew.sh/formula/jq)
+- [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html#getting-started-install-instructions)
+
+#### Setup AWS Profile
```

Comment:
Let's nest part this under `subspace init` instructions at line 45 since it's part of the init process